product-scenarios/4-gateway/4.1-Extend-the-reach-for-existing-or-legacy-applications/4.1.1-Expose-back-ends-exposed-over-HTTP-as-SOAP-service/4.1.1.1-Expose-a-SOAP-service-as-SOAP-service-using-proxy-service/pom.xml

```diff
@@ -47,11 +47,18 @@
                     <systemProperties>
                         <property>
                             <!-- Common resource directory (eg: Key stores, certificates, etc.)-->
-                            <name>test.resources.dir</name>
+                            <name>common.resources.dir</name>
```

Comment:
Can we move thise to a more common place, may be to scenarios-commons?

Reply:
It is already moved to a ScenarioTestBase class. But we should be able to build each test case individually right? In that case, these properties will be needed.